src/screens/nano/NanoContractDetail.js

```diff
@@ -144,6 +144,35 @@ function NanoContractDetail(props) {
     });
   }
 
+  const renderBalances = () => {
+    return Object.entries(ncState.balances).map(([token, data]) => {
```

Comment:
suggestion(non-blocking): Rename `token` to `tokenUid` or `uid`, to make explicit that it is just a string and not an object.

Reply:
Done https://github.com/HathorNetwork/hathor-explorer/pull/287/commits/97222fc0248cf23d79765f40ef45a80154829a75